src/control/lib/spdk/ctests/SConscript

```diff
@@ -20,8 +20,8 @@ def scons():
     unit_env.AppendUnique(CPPPATH=[nvme_lib_src, spdk_incl_src])
 
     # SPDK related libs
-    libs = ['spdk_env_dpdk', 'spdk_thread', 'spdk_bdev', 'spdk_copy']
-    libs += ['rte_mempool', 'rte_mempool_ring', 'rte_bus_pci']
+    libs = ['spdk_env_dpdk', 'spdk_thread', 'spdk_bdev', 'spdk_accel']
+    libs += ['rte_mempool', 'rte_mempool_ring', 'rte_bus_pci', 'rte_power']
     libs += ['rte_pci', 'rte_ring', 'rte_mbuf', 'rte_eal', 'rte_kvargs']
     libs += ['spdk_bdev_aio', 'spdk_bdev_nvme', 'spdk_bdev_malloc']
     libs += ['spdk_conf', 'spdk_blob', 'spdk_nvme', 'spdk_util']
```

Comment:
Why so many more libs are required for 'spdk/ctests'?

Reply:
this just needs updating